app/models/partner.rb

```diff
@@ -22,7 +22,7 @@ class Partner < ApplicationRecord
   has_many :requests, dependent: :destroy
 
   validates :organization, presence: true
-  validates :name, :email, presence: true, uniqueness: true
```

Comment:
https://guides.rubyonrails.org/active_record_validations.html#uniqueness

Instead of removing the constraint entirely (it's either out-of-scope for this PR, or if it's related to the seeds, we shouldn't be changing model constraints just to make the seeds work better), you can scope it to the organization and that would probably be sufficient?

Reply:
Yes, if we can scope it to the organization then we can restore the uniqueness test. Do you know how to do that? I tried adding `, scope: organization` and  when I ran the seeder I got `ArgumentError: Unknown validator: 'ScopeValidator'`.

IMO, as it is currently (scoped globally), the validation is not a legitimate one, so I'm not that reluctant to remove it, at least temporarily.

Reply:
I do realize that removing the constraint and readding it later would possibly require cleaning up data and is something we'd want to avoid. I removed the constraint because it was preventing me from adding the same partner data to 2 org's. If you like I could kludge it and add 'pdx' or 'sf' to the data so they would not be duplicates and then restore the constraint I removed. I do think we should address the global scope issue ASAP though.

Reply:
I found what I believe to be the correct incantation and the seeding appears to work with the scoped uniqueness validation:

`  validates :name, :email, presence: true, uniqueness: { scope: :organization }
`
I've pushed this change.